core/capabilities/remote/dispatcher.go

```diff
@@ -168,6 +179,11 @@ func (d *dispatcher) receive() {
 			d.lggr.Info("stopped - exiting receive")
 			return
 		case msg := <-recvCh:
+			if !d.rateLimiter.Allow(msg.Sender.String()) {
+				d.lggr.Debugw("rate limit exceeded, dropping message", "sender", msg.Sender)
+				// TODO: do we just drop the message, or do we need to respond with an error?
```

Comment:
Just drop. We don't want to respond as this is a protection against spammy nodes.